### ISSUE TITLE
meshctrl not working: 502 Websocket issue

### ISSUE THREAD

**Issue by abacusea** [Ylianst/MeshCentral#3807]
<img width="542" alt="meshctrl err" src="https://user-images.githubusercontent.com/55495252/160295120-2c29a673-7591-4f4d-9c02-41a44c9201e1.PNG">

502 Server issue return whenever I try to run meshctrl code:

```
node node_modules\meshcentral\meshctrl --url wss://mesh.arbiit.com:443/ --loginuser #### --loginpass **********
```

Server nginx.conf updated with this code:
```
worker_processes 1;
events {
    worker_connections 1024;
}
http {
 # HTTP server. In this example, we use a wildcard as server name.
 server {
 listen 80;
 server_name mesh.arbiit.com;
    location / {
    proxy_pass https://mesh.arbiit.com/;
    proxy_http_version 1.1;

    # Inform MeshCentral about the real host, port and protocol
    proxy_set_header X-Forwarded-Host $host:$server_port;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    }
 }
 # HTTPS server. In this example, we use a wildcard as server name.
 server {
 listen 443 ssl;
 server_name _;
 # MeshCentral uses long standing web socket connections, set longer timeouts.
 proxy_send_timeout 330s;
 proxy_read_timeout 330s;
 # We can use the MeshCentral generated certificate & key
 ssl_certificate /etc/letsencrypt/live/arbiit.com/fullchain.pem;
 ssl_certificate_key /etc/letsencrypt/live/arbiit.com/privkey.pem;
 ssl_session_cache shared:WEBSSL:10m;
 ssl_ciphers HIGH:!aNULL:!MD5;
 ssl_prefer_server_ciphers on;
 location / {
    proxy_pass https://mesh.arbiit.com:4430/;
    proxy_http_version 1.1;

    # Allows websockets over HTTPS.
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "upgrade";
    proxy_set_header Host $host;
    # Inform MeshCentral about the real host, port and protocol
    proxy_set_header X-Forwarded-Host $host:$server_port;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    }
 }
}
```

**Comment by Ylianst** [comment]
Ok, I just fixed meshctrl.js to display a more friendly error message:

```
node meshctrl --url wss://mesh.arbiit.com:443/ --loginuser abc --loginpass def listdevices
Unable to connect to wss://mesh.arbiit.com:443/control.ashx
```

I tried connecting to your server and before this fix, got the same error you got. But, using a browser I see this page:

![image](https://user-images.githubusercontent.com/1319013/160467215-78a59fe2-d529-4ca5-9006-f6b3b1e20460.png)

So, I imagine your server is not currently working. Fix this and meshctrl.js should work.
